### ISSUE TITLE
Access to this attachment is blocked

### ISSUE THREAD

**Issue by fruitloop** [ecederstrand/exchangelib#263]
Hi,

I'm generating an XML File and attach that to a calendar event as well as an email.

However lately I'm getting the following message in my Office365 webmail:

`Access to this attachment is blocked. Recipients may not be able to view the attachment, either.`

**Comment by ecederstrand** [comment]
Maybe this is caused by some server policy on attachments. You could try to create an event with the attachment using Office365. If that works, then you can use exchangelib to inspect the item and find out how it's different from the item you created through exchangelib.

**Comment by fruitloop** [comment]
Why didn't I think of that in the first place ;)

I'll try that.

**Comment by Benco1** [comment]
By "lately", what do you mean? @fruitloop

**Comment by fruitloop** [comment]
Well I found the problem. Outlook Web App (which I started using a few weeks ago since my android exchange app somehow removed the support to share attachments from calendar entries) blocks XML Extensions by default.

However it seems I'll have to use Remote Power Shell to change that. So I'll have to find another way to solve this problem since I have no access to a Remote Power Shell at the moment.

It is not however a problem of exchangelib since in Outlook I can open the attachment just fine.
